src/test/java/se/sundsvall/comfactfacade/api/SigningResourceTest.java

```diff
@@ -35,152 +35,145 @@ class SigningResourceTest {
 	@Autowired
 	private WebTestClient webTestClient;
 
-
 	@Test
-	void createSigningRequest() {
+	void getSigningRequests() {
+
 		// Arrange
-		final var signingRequest = new TestUtil.SigningRequestBuilder().build();
+		when(signingServiceMock.getSigningRequests()).thenReturn(List.of(new SigningInstance(), new SigningInstance()));
+
 		// Act
-		webTestClient.post()
+		final var result = webTestClient.get()
 			.uri("/signings")
-			.contentType(APPLICATION_JSON)
-			.bodyValue(signingRequest)
 			.exchange()
-			.expectStatus().isOk();
+			.expectStatus().isOk()
+			.expectBodyList(SigningInstance.class)
+			.returnResult()
+			.getResponseBody();
 
 		// Assert
-		verify(signingServiceMock).createSigningRequest(any(SigningRequest.class));
+		verify(signingServiceMock).getSigningRequests();
+		assertThat(result).hasSize(2);
+
 	}
 
+
 	@Test
-	void cancelSigningRequest() {
+	void createSigningRequest() {
+
 		// Arrange
-		final String signingId = "someSigningId";
+		final var signingRequest = SigningRequest.builder()
+			.withSignatory(Party.builder().build())
+			.withInitiator(Party.builder().build())
+			.withDocument(Document.builder()
+				.withFileName("someFileName")
+				.withMimeType("someMimeType")
+				.withContent("someContent")
+				.build())
+			.build();
+
+		when(signingServiceMock.createSigningRequest(any(SigningRequest.class)))
+			.thenReturn("someSigningId");
 
 		// Act
-		webTestClient.delete()
-			.uri("/signings/{signingId}", signingId)
+		webTestClient.post()
+			.uri("/signings")
+			.contentType(APPLICATION_JSON)
+			.bodyValue(signingRequest)
 			.exchange()
-			.expectStatus().isNoContent();
+			.expectStatus()
+			.isCreated()
+			.expectHeader()
+			.location("/signings/someSigningId");
 
 		// Assert
-		verify(signingServiceMock).cancelSigningRequest(signingId);
+		verify(signingServiceMock).createSigningRequest(any(SigningRequest.class));
```

Comment:
Why not use the actual request object to validate that correct data is sent to service (i.e `verify(signingServiceMock).createSigningRequest(signingRequest);`)? 

Comment goes for all verifcations where `any(...class)` is used.